var/spack/repos/builtin/packages/conduit/package.py

```diff
@@ -375,6 +377,19 @@ def hostconfig(self):
             if flags:
                 cfg.write(cmake_cache_entry("BLT_EXE_LINKER_FLAGS", flags, description))
 
+        #######################
+        # Examples/Utilities
+        #######################
+        if "+examples" in spec:
+            cfg.write(cmake_cache_entry("ENABLE_EXAMPLES", "ON"))
+        else:
+            cfg.write(cmake_cache_entry("ENABLE_EXAMPLES", "OFF"))
```

Comment:
Looks good.  Given the way this looks, the following would be tighter:

```suggestion
        cfg.write(cmake_cache_entry("ENABLE_EXAMPLES", "ON" if "+examples" in spec else "OFF"))
```

Reply:
I would prefer commit style b/c its easier for my brain to parse and understand when scanning vs embedding an if in a function call. Is that ok?